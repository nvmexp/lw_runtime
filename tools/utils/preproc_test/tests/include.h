line 1
#include "simple.h"
line 2
#include <simple.h>
line 3
#include "subdir/subinclude.h"
line 4
