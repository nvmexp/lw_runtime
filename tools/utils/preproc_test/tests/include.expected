#line 1 "include.h"
line 1

#line 1 "simple.h" 1
Hello, World!
#line 3 "include.h" 2
line 2

#line 1 "simple.h" 1
Hello, World!
#line 5 "include.h" 2
line 3

#line 1 "subdir/subinclude.h" 1

#line 1 "subdir/simple.h" 1
Simple header - include test.
#line 2 "subdir/subinclude.h" 2
#line 7 "include.h" 2
line 4
